### DISCUSSION TITLE
Download issues

### DISCUSSION THREAD

**Discussion by israpps** [israpps/FreeMcBoot-Installer#25] [Q&A]
Please report here if any download is down

**Comment by wavekbbs** [comment]
下載完，解不開

**Comment by davoodtitan** [comment]
I installed version 1.96 and I had a few problems. First, version 1.96 supports exFAT, but there is a problem. When you go into the FC Boot settings, for example, if you want to bring a program to the home screen, it does not read the flash memory. The flash memory must be in FAT32 format.

**Comment by israpps** [comment]
nothing I can do about that sadly

FreeMcBoot is a dead end in terms of advancements. the program is abandoned, closed source and the developer went on with life

**Reply by davoodtitan** [reply]
I just pulled the Sony PlayStation 2 out of the basement 😆

**Comment by cagatay179** [comment]
i cant access freemcboot it does finishes downloading but not get in the menu
